<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>SpellFindBase</class>
  <widget class="QDialog" name="SpellFind" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>354</width>
        <height>97</height>
      </rect>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Find ASpell</string>
    </property>
    <property name="sizeGripEnabled" stdset="0" >
      <bool>true</bool>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item rowspan="1" row="0" column="0" colspan="3" >
        <widget class="QLabel" name="lblPath" >
          <property name="sizePolicy" stdset="0" >
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>7</vsizetype>
            </sizepolicy>
          </property>
          <property name="text" stdset="0" >
            <string/>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="1" column="1" >
        <widget class="QPushButton" name="btnCancel" >
          <property name="text" stdset="0" >
            <string>&amp;Cancel</string>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <spacer name="Spacer3" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="1" column="2" >
        <spacer name="Spacer4" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
